cognee/api/v1/search/search.py

```
@@ -78,16 +78,21 @@
 
     search_tasks = []
 
+    if user is None:
```

Ruff

Undefined name `user`

(F821)